api/src/main/java/io/minio/messages/ListBucketResultV1.java

```diff
@@ -26,22 +30,25 @@
  */
 @Root(name = "ListBucketResult", strict = false)
 @Namespace(reference = "http://s3.amazonaws.com/doc/2006-03-01/")
-public class ListBucketResultV1 extends ListBucketResult {
+public class ListBucketResultV1 extends ListObjectsResult {
   @Element(name = "Marker", required = false)
   private String marker;
 
   @Element(name = "NextMarker", required = false)
   private String nextMarker;
 
-  /** Returns continuation token. */
-  @Override
-  public String continuationToken() {
+  @ElementList(name = "Contents", inline = true, required = false)
+  private List<Contents> contents;
+
+  public String marker() {
     return marker;
   }
 
-  /** Returns next continuation token. */
-  @Override
-  public String nextContinuationToken() {
+  public String nextMarker() {
     return nextMarker;
   }
+
+  public List<Item> contents() {
```

Comment:
Add the `@Override` annotation, and change the implementation to call `emptyIfNull()` if you accept my suggestion in `ListObjectsResult`